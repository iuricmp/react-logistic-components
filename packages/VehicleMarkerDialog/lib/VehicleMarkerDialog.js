"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;

var _react = _interopRequireDefault(require("react"));

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }

function VehicleMarkerDialog() {
  return _react["default"].createElement("h1", null, "VehicleMarkerDialog 2");
}

var _default = VehicleMarkerDialog;
exports["default"] = _default;