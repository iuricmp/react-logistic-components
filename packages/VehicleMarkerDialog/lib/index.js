"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
Object.defineProperty(exports, "default", {
  enumerable: true,
  get: function get() {
    return _VehicleMarkerDialog["default"];
  }
});

var _VehicleMarkerDialog = _interopRequireDefault(require("./VehicleMarkerDialog"));

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }