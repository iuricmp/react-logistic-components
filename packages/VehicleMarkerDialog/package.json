{
  "name": "@react-logistic-components/vehicle-marker-dialog",
  "version": "0.0.4-alpha.0",
  "description": "Vehicle marker dialog",
  "main": "lib/index.js",
  "module": "src/index.js",
  "author": "React Logistic Components Team",
  "peerDependencies": {
    "react": "^16.9",
    "react-dom": "^16.9"
  }
}
