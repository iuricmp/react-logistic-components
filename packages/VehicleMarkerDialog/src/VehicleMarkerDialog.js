import React from 'react';

function VehicleMarkerDialog() {
  return <h1>VehicleMarkerDialog 2</h1>
}

export default VehicleMarkerDialog
