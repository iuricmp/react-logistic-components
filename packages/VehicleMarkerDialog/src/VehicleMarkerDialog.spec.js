import React from 'react';
import renderer from 'react-test-renderer';
import VehicleMarkerDialog from './VehicleMarkerDialog'

it('renders correctly', () => {
  const tree = renderer
    .create(<VehicleMarkerDialog />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
