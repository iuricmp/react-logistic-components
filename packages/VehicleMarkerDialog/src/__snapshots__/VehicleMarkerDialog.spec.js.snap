// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<h1>
  VehicleMarkerDialog 2
</h1>
`;
